### DISCUSSION TITLE
Setting column width breaks other columns

### DISCUSSION THREAD

**Discussion by DrMalchev** [UI5/webcomponents-react#6111] [General]
If I set no widths, the columns are auto-spread to fill the available space. If I want to set individual width for a column, the table does not rearrange to fill the space but shows empty space on the right (if custom width is less than the default) or scroller (if width is more than default).

I am sure I miss something. Thnaks in advance for the help!

**Comment by DrMalchev** [comment]
I found out that width set in absolute value doesn't break the component.
{
    Header: "Url",
    accessor: "URL",
    headerTooltip: "Url",
width:"85px"
} will break

and

{
    Header: "Url",
    accessor: "URL",
    headerTooltip: "Url",
width:85
} will NOT break

**Reply by MarcusNotheis** [reply]
Hi @DrMalchev,
good to hear that you were already able to solve your issue. 
You're totally right, the column options of the `AnalyticalTable` require the `width` properties to be of type number: https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable--docs
